model-optimizer/extensions/front/onnx/loop_ext.py

```diff
@@ -29,10 +29,14 @@ def extract(cls, loop_node):
 
         # create a Graph object for the body and take graph attributes from the main graph
         body_graph = Graph()
-        main_graph_attrs_copy = copy.deepcopy(main_graph.graph)
-        del main_graph_attrs_copy['tensor_mapping']
+        main_graph_attrs_copy = {}
+        for attr_key in main_graph.graph.keys():
```

Comment:
probably it can be replaced with:
`for attr_key, attr_value in main_graph.graph.items()`